tools/monitoring/mysql/error_logger.go

```diff
@@ -21,47 +21,30 @@ import (
 	"time"
 
 	"github.com/knative/test-infra/shared/mysql"
-	"github.com/knative/test-infra/tools/monitoring/config"
-	"github.com/knative/test-infra/tools/monitoring/log_parser"
 )
 
-const (
-	logInsertStmt = `
-	INSERT INTO ErrorLogs (
-		ErrorPattern, ErrorMsg, JobName, PRNumber, BuildLogURL, TimeStamp
-		) VALUES (?,?,?,?,?,?)`
-)
-
-// PubsubMsgHandler adds record(s) to ErrorLogs table in database,
-// after parsing build log and compares the result with config yaml
-func PubsubMsgHandler(db *sql.DB, configURL, buildLogURL, jobname string, prNumber int) error {
-	config, err := config.ParseYaml(configURL)
-	if err != nil {
-		return err
-	}
-
-	errorLogs, err := log_parser.ParseLog(buildLogURL, config.CollectErrorPatterns())
-	if err != nil {
-		return err
-	}
+// DB holds an active database connection created in `config`
+type DB struct {
```

Comment:
why do we need another struct? We can store *sql.db on DBConfig and have mthods on DBConfig?

Reply:
I cannot extend type in other packages. Since `InsertErrorLog` is a monitoring database specific method, I created a separate struct for it.

Reply:
Are you actually using any properties/method from DBConfig? If not, why do you include it as part of the DB here?

Reply:
I think I was debugging the host info when connecting to the db. But since that's not needed anymore. Removed the config from the struct.